data/update_script.sh.erb

```diff
@@ -5,6 +5,8 @@ set -o pipefail
 
 APP_TARBALL="$1"
 APP_PATH="<%= locals.remote_app_path %>"
+PULLPREVIEW_LAUNCH_COMMAND="<%= locals.custom_launch_command %>"
+PULLPREVIEW_ENV_VARS="<%= locals.custom_env_vars %>"
```

Comment:
For env vars, I strongly suggest you simply output them in a `.env` file (or any other custom env file that can be read by your compose definition) during the action execution. That file will be [automatically read](https://docs.docker.com/compose/environment-variables/#the-env-file) by docker-compose when booting.

Reply:
yeah, I know of the option to passing in .env files to docker-compose. The problem is that I'm having a handful of environment variables I want to set independent of that like for example I have a dynamic docker image tag in my docker-compose file or I'd like to have the /app/bin directory in the PATH.